internal/oem/oem.go

```diff
@@ -232,6 +233,25 @@ alias gsutil="(docker images google/cloud-sdk || docker pull google/cloud-sdk) >
 		name:  "file",
 		fetch: file.FetchConfig,
 	})
+	configs.Register(Config{
+		name:  "sakuracloud",
+		fetch: sakuracloud.FetchConfig,
+		baseConfig: types.Config{
+			Systemd: types.Systemd{
+				Units: []types.SystemdUnit{{
+					Enable: true,
+					Name:   "coreos-metadata-sshkeys@.service",
+					DropIns: []types.SystemdUnitDropIn{{
+						Name: "00-sakuracloud.conf",
+						Contents: `[Service]
+ExecStart=
+ExecStart=/usr/bin/coreos-metadata --attributes /etc/environment --cmdline --hostname /etc/hostname --network-units /etc/systemd/network --ssh-keys=%i
```

Comment:
Instead of overriding `coreos-metadata-sshkeys@.service`, can you just create a new unit (e.g. `coreos-sakuracloud-network.service`) to generate the network units?

Will this network config ever change throughout the lifetime of the machine?